desktop/src/main/java/bisq/desktop/main/content/user/user_profile/UserProfileView.java

```diff
@@ -158,11 +159,7 @@ protected void onViewAttached() {
             }
         });
 
-        deleteButton.setOnAction(e -> {
-            if (comboBox.validate()) {
-                controller.onDeleteProfile();
-            }
-        });
+        deleteButton.setOnAction(e -> onDeleteButtonPressed());
```

Comment:
I understand the reasons why to call the method on the View as the validators are in the view or inside the components. But we break the Bisq2 MVC concept that the View is not doing any logic (beside layout logic) and the controller is responsible for such. 
If we create and reference the validators in the controller and pass it to the view we could do that in the controller. I am aware that this is also not a super nice solution but I would consider it better. Validation is at the end a task for the controller.
But (if you agree) no need to do that change in that PR, could be a follow up PR.

Reply:
You're bringing up a good point. I need to think more about this. I copied the validation framework from JFoenix as it was suggested in the requirements of https://github.com/bisq-network/bisq2/issues/883. So the validators are coupled to the UI components - not sure if creating them in the controller would be an improvement unless we rethink the whole validation framework. I don't think it's a big problem right now as it's well encapsulated and should be easy to replace by another solution in the future. I'd suggest we give it a thought after we complete the 2.0.0 milestone. Maybe create an issue for that?